website/docs/r/user.html.markdown

```diff
@@ -46,7 +46,7 @@ When a user is removed from all teams completely, they will inherit whatever per
 If a user is removed from all their teams, it will probably be necessary to run `terraform apply` a second time
 to update the users permissions from their old team permissions to new user-specific permissions.
 
-See [the NS1 API docs](https://ns1.com/api#getget-all-account-users) for an overview of permission semantics.
+See https://help.ns1.com/hc/en-us/articles/360024409034-Managing-user-permissions for an overview of user permission settings.
```

Comment:
This should probably be a link, as before. ie

```
See [this Help Center article](https://help.ns1.com/hc/en-us/articles/360024409034-Managing-user-permissions) for an overview of user permission settings.
```

Reply:
yea totally, that makes sense. Should I make the edit and create another PR? or can I make the change here somehow